bitnami/minio/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 3.4.21
+version: 3.4.22
```

Comment:
Please bump the minor version instead since you're including a new parameter.

Reply:
Done, thank you